themes/developers/dev/assets/sass/utilities/variables/_typography.scss

```diff
@@ -4,6 +4,7 @@ $font-family--lato: 'Lato', sans-serif;
 
 // Base Typography
 $html--font-size: 62.5%;
+$rem-baseline: 10px;
```

Comment:
@rgadon107 I'd suggest:

1. moving the rem fallback parameter here too, ie to keep all of the REM configuration parameters together
2. Moving both of them up to the top of this Sass partial and adding a comment for them.

Reply:
@hellofromtonya Done.